server/app/services/settings/AbstractSettingsManifest.java

```diff
@@ -32,7 +32,7 @@ public AbstractSettingsManifest(Config config) {
    * Returns a map containing the names of the settings in the "Feature Flags" section mapped to
    * their current values.
    */
-  public ImmutableSortedMap<String, Boolean> getAllFeatureFlagsSorted(Http.Request request) {
+  public ImmutableSortedMap<String, Boolean> getAllFeatureFlagsSorted(Http.RequestHeader request) {
```

Comment:
💁🏻  this PR makes a small type change for the request object, changing the type from `Http.Request` to `Http.RequestHeader`. The former is a concrete type that Play passes to controller actions. Some views are not rendered by controllers, such as the custom views for error responses, and are instead passed a `Http.RequestHeader`. Since ``Http.Request` implements `Http.RequestHeader` header we can specify the latter in both places.